src/hal/utils/scope_disp.c

```diff
@@ -556,12 +557,12 @@ static double snap(int y) {
 }
 
 static void left_drag(int dy, int y, GdkModifierType state) {
+    if(vert->selected <= 0) return;
+
```

Comment:
This doesn't build -- needs to be moved back down.

Reply:
weird, I thought we switched to a modern standard that supported interspersing declarations & other statements. I guess that's in master only, not 2.9.

It has to be before one of the array indexes of `vert->selected`, otherwise there's still undefined behavior (even though it's probably OK on all normal computers)

Reply:
oh, oops, OK. :blush: